prj.conf

```diff
@@ -4,104 +4,153 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
+# This sample uses Kconfig.defaults to set options common for all
+# samples. This file should contain only options specific for this sample
+# or overrides of default values.
+
 # Enable CHIP
 CONFIG_CHIP=y
 CONFIG_CHIP_ENABLE_PAIRING_AUTOSTART=y
 CONFIG_CHIP_PROJECT_CONFIG="src/chip_project_config.h"
+# 32774 == 0x8006 (example lock-app)
+# 0x4E54: nrfconnect bridge (not used due to missing DAC)
+CONFIG_CHIP_DEVICE_PRODUCT_ID=32774
 CONFIG_STD_CPP14=y
 
-# General networking settings
-CONFIG_NETWORKING=y
-CONFIG_NET_IPV6_NBR_CACHE=n
-CONFIG_NET_IPV6_MLD=n
-CONFIG_NET_SOCKETS_POLL_MAX=5
-
-# Enable OpenThread MTD
-CONFIG_NET_L2_OPENTHREAD=y
+# Add support for LEDs and buttons on Nordic development kits
+CONFIG_DK_LIBRARY=y
 
+# OpenThread configs
+CONFIG_OPENTHREAD_NORDIC_LIBRARY_MTD=y
 CONFIG_OPENTHREAD_MTD=y
 CONFIG_OPENTHREAD_FTD=n
-CONFIG_OPENTHREAD_SLAAC=y
-CONFIG_OPENTHREAD_MTD_NETDIAG=y
-CONFIG_OPENTHREAD_MANUAL_START=y
-CONFIG_OPENTHREAD_THREAD_STACK_SIZE=6144
-CONFIG_OPENTHREAD_NUM_MESSAGE_BUFFERS=64
 
+# Default OpenThread network settings
```

Comment:
I think this configuration is not required. It used to be supported to form the default Thread network without commissioning, but the support was dropped.

Reply:
removed.